GNUmakefile

```diff
@@ -43,6 +48,11 @@ CFLAGS      += -qopenmp
 CFLAGS      += -I./
 CFLAGS		+= -fp-model precise
 
+ifeq ($(MKL),1)
+	CFLAGS += -mkl
+	CFLAGS += -DMKL
```

Comment:
the compiler flag should be more unique in order to avoid collision with MKL itself